### PR TITLE
[AN-4417] Adds likers list on tablet

### PR DESCRIPTION
**Pull request by shumengye**
#### Ticket
https://wearezeta.atlassian.net/browse/AN-4417
#### APK
[Download build #7519](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7519/artifact/build/artifact/wire-dev-PR122-7519.apk)
[Download build #7520](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7520/artifact/build/artifact/wire-dev-PR122-7520.apk)
[Download build #7521](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7521/artifact/build/artifact/wire-dev-PR122-7521.apk)
[Download build #7523](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7523/artifact/build/artifact/wire-dev-PR122-7523.apk)

### CODE REVIEW

**Comment by shumengye** [comment]
build again

**Comment by hakonbo** [comment]
build again

File: app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/footer/views/FooterLikeDetailsLayout.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 public class FooterLikeDetailsLayout extends LinearLayout {
 
-    private static final int NUM_LIKE_USERS_TO_SHOW_AVATARS = 3;
+    private static final int NUM_LIKE_USERS_TO_SHOW_AVATARS = 0;
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/footer/views/FooterLikeDetailsLayout.java]: I guess this one was included by mistake?

Comment (shumengye) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/views/row/footer/views/FooterLikeDetailsLayout.java]: yes, left over from testing

**Pull request merged**